go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/briandowns/spinner v1.19.0
 	github.com/buildpacks/pack v0.21.0
+	github.com/cavaliergopher/grab/v3 v3.0.1
```

Comment:
This change is isolated in [this commit](https://github.com/superfly/flyctl/pull/1830/commits/9532519491336edef9d96c8d95b23463a1189c0c), it should have been part of https://github.com/superfly/flyctl/pull/1817